src/components/Courses/Courses.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+import CourseCard from '../CourseCard/CourseCard';
+import CreateCourse from '../CreateCourse/CreateCourse';
+import { Button } from '../Button/Button';
+import { mockedCoursesList } from '../../utils/constants';
+
+const CoursesContainer = styled.div`
+	width: 80%;
+	margin: 20px auto;
+	border: 2px solid red;
+`;
+
+const CoursesTop = styled.div`
+	width: 80%;
+	margin: 20px auto;
+	display: flex;
+	justify-content: space-between;
+`;
+
+const StyledSearch = styled.div`
+	width: 30%;
+	height: 30%;
+`;
+
+const Courses = () => {
+	const [search, setSearch] = useState('');
+	const [courseCreating, setCourseCreating] = useState(false);
+	const textInput = useRef(null);
+
+	const goSearch = (event) => {
+		event.preventDefault();
+		setSearch(textInput.current.value);
```

Comment:
usually, in react we use one way data-binding
In this particular case it's ok to use ref, but this approach is not common in react

Reply:
You no longer need textInput ref: `value` can be taken from event (`event.target.value`)